runtime/onert/core/src/compiler/StaticShapeInference.cc

```diff
@@ -25,6 +25,64 @@ namespace onert
 namespace compiler
 {
 
+bool StaticShapeInferer::infer(const ir::OpSequence &op_seq)
+{
+  bool has_dynamic_tensor = false;
+
+  for (const auto &operation_idx : op_seq.operations())
+  {
+    auto &op = _operations.at(operation_idx);
+    auto opcode = op.opcode();
+
+    _return_has_dynamic_tensor = false; // this is used as a return value inside operation's visit()
```

Comment:
(optional) It would be nice to change this member variable to a local variable.

Reply:
OK. I'll fix it.

Reply:
It cannot be changed to local variable because this variable is used in visitors.

Reply:
Okay. I did miss out it.